.github/dependabot.yml

```diff
@@ -1,11 +1,18 @@
 version: 2
 updates:
-- package-ecosystem: gomod
-  directory: "/"
-  schedule:
-    interval: weekly
-  open-pull-requests-limit: 10
-- package-ecosystem: "github-actions"
-  directory: "/"
-  schedule:
-    interval: weekly
+  # Go
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    pull-request-branch-name:
+      separator: "-"
+    open-pull-requests-limit: 4
```

Comment:
If we're doing weekly PRs and grouping then I don't think there's much value in a open PR limit (it should never really be above 1) and :shrug: on the branch name stuff, just let it use the default

```suggestion
```